### PR TITLE
[wip] fix issue #16 by running yield tests inline

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
while degrading reporting this one fixes setupstate

reporting should be fixed if/when introducing subtests
an alternative would be to completely drop yield test support in core

### CODE REVIEW

**Comment by nicoddemus** [comment]
Changed milestone to `3.0` so we can discuss this on the sprint. :grin:

File: testing/python/collect.py [hunk 1]
```diff
@@ -107,197 +107,7 @@ def __getattr__(self, name):
 
 
 class TestGenerator:
```
Comment (nicoddemus) [inline testing/python/collect.py]: Let's remove the class declaration as well

Comment (nicoddemus) [inline testing/python/collect.py]: On second thought, we should have new tests for the new behavior

**Comment by RonnyPfannschmidt** [comment]
closing due to sprint discussions wanting a different solution

**Pull request closed without merging**